netaddr/tests/eui/test_eui.py

```diff
@@ -158,6 +158,7 @@ def test_eui_oui_information():
 def test_oui_constructor():
     oui = OUI(524336)
     assert str(oui) == '08-00-30'
+    assert hash(oui) == 524336
```

Comment:
We need a real-world hash-capability demonstration here. If the original issue is about using `OUI` objects as dictionary keys let's use that to check if it works, we shouldn't be hardcoding specific hash values (those can't be depended on etc.).

Reply:
Fair enough, let me know if you want me to drop the `hash` asserts in the new one.  Broke it out into a different func to not clutter the `test_oui_constructor`.

Reply:
Thank you! Yes, please drop those two asserts and I'm happy to merge this.